mkt/site/utils.py

```diff
@@ -265,11 +265,11 @@ def __iter__(self):
             return iter([])
 
         chunk = 4096
-        fp = open(self.path, 'rb')
+        fp = storage.open(self.path, 'rb')
```

Comment:
The entire `HttpResponseSendFile` class won't work :( There are legitimate uses for it though - things that do status/permission checks before serving a package for instance - so I wonder what we'll replace it with. Not sure a redirect will be enough (I worry about security and compatibility with older FxOS devices)